src/app/markdown/markdown.component.scss

```diff
@@ -0,0 +1,70 @@
+@import "../../assets/stylesheets/_base";
+
+[contenteditable]:focus {
+    outline: 0px solid transparent;
+}
+
+.tab-btn {
+  cursor: pointer;
+}
+
+
+.editor-container {
+    background-color: $color-pf-black-200;
+
+    .editor-markdown {
+      font-size: 15px;
+    }
+
+    .editor-preview {
+      h1 {
+        margin-top: 0px;
+      }
+    }
+
+    .editor {
+      P {
+        margin: 0px;
+      }
+
+      .editor-box {
+        min-height: 80px;
+        width: 100%;
+      }
+
+      .edit-icon {
+        position: relative;
+        left: calc(100% + 10px);
+        top: 10px;
+        cursor: pointer;
+        opacity: 0;
+      }
+
+      &:hover {
+        .edit-icon {
+          opacity: 1;
+        }
+      }
+
+      background-color: $color-pf-white;
+      padding-right: 30px;
+      background-clip: content-box;
+      border: 1px solid $color-pf-black-500;
+    }
+
+    .show-less {
+      max-height: 300px;
+      overflow: hidden;
+    }
+
+    .active {
+      border: 1px solid $color-pf-blue-400;
+    }
+}
+
+.actions {
+  margin-top: 8px;
```

Comment:
indentation of items inside of `.actions {` is different than above. All should be 2 spaces.